libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -207,23 +263,29 @@ export class DrivingLicenseService {
     nationalId: User['nationalId'],
     input: NewDrivingLicenseInput,
   ): Promise<NewDrivingLicenseResult> {
-    const response = await this.drivingLicenseApi.newDrivingLicense({
-      authorityNumber: input.juristictionId,
-      needsToPresentHealthCertificate: input.needsToPresentHealthCertificate
-        ? 1
-        : 0,
-      personIdNumber: nationalId,
-    })
+    const response: unknown = await this.drivingLicenseApi.apiOkuskirteiniApplicationsNewCategoryPost(
+      {
+        category: 'B',
+        postNewFinalLicense: {
+          authorityNumber: input.juristictionId,
+          needsToPresentHealthCertificate: input.needsToPresentHealthCertificate
+            ? 1
```

Comment:
Same here, I guess it's a boolean, but it's easier sometimes to have something like this: (just an example)

```ts
enum HealhCertificate {
  TRUE = 1,
  FALSE = 0,
}
```